src/environments/environment.ts

```diff
@@ -30,7 +30,7 @@ export const environment = {
 
 export const authConfig: AuthConfig = {
   // Url of the Identity Provider
-  issuer: 'https://identity.ret-us-l.rnd.live.backbaseservices.com/auth/realms/backbase',
+  issuer: 'https://identity.dev.sdbx.live.backbaseservices.com/auth/realms/retail',
```

Comment:
can you also check this https://github.com/Backbase/golden-sample-app/blob/master/docker-compose.yml#L16 and update if possible